sonar-kotlin-gradle/src/test/samples/non-compiling/DependencyGroupedCheckSample.kts

```diff
@@ -0,0 +1,110 @@
+dependencies {
```

Comment:
While currently, it's okay to have a 'non-compiling' Gradle examples, let's aim to have valid Gradle files to simplify the process once we have a semantic model available